docker/dockerfiles/Dockerfile

```diff
@@ -1,154 +1,215 @@
-# Based on Code.org CircleCI-dependencies Dockerfile in .circle directory
 # Pushed to Docker Hub at codedotorg/code-dot-org:<version>
-FROM ubuntu:18.04
-
-USER root
-
-# set timezone to UTC by default
-RUN ln -sf /usr/share/zoneinfo/Etc/UTC /etc/localtime
```

Comment:
has this functionality been removed? I'm not seeing an equivalent in the new version

Reply:
This is handled using the TZ env var at the top instead, which is prettier and "more docker" in the sense that it allows you to vary the TZ for a container at start (not something we want to do on drone, but makes the dockerfile more re-usable for e.g. dev):

```
ENV \
    LANG=C.UTF-8 \
    TZ=Etc/UTC \
```